source/sdk/lang/Memory.ooc

```diff
@@ -1,53 +1,17 @@
-
-// as goofy as it sounds, memory-related routines are actually in string.h
-// yay C. (Also, Google Native Client doesn't have memory.h)
 include string
 version(windows) { include malloc }
```

Comment:
Oh wow, so you can use version blocks around includes, I thought it does not work at all. Nice to know.